.github/workflows/python_tests.yaml

```diff
@@ -0,0 +1,58 @@
+name: Python Tests
+on:
+  push:
+    branches:
+      - develop
+      - 'release/*'
+  pull_request:
+    branches:
+      - develop
+      - 'release/*'
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash -leo pipefail {0}
+
+jobs:
+  srw_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install Micromamba and srw_app environment
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-file: environment.yml
+          environment-name: srw_app
+          init-shell: bash
+          cache-downloads: true
+          cache-environment: true
+
+      - name: Checkout externals
+        run: |
+          ./manage_externals/checkout_externals ufs-weather-model workflow-tools
+
+      - name: Lint the python code
+        run: |
+          micromamba activate srw_app
+          export PYTHONPATH=$(pwd)/ush:$(pwd)/ush/python_utils/workflow-tools:$(pwd)/ush/python_utils/workflow-tools/src
```

Comment:
@christinaholtNOAA Just wanted to check that the repeated env activation and PYTHONPATH exports are needed for each task. GHA doesn't carry over that info? Or is there a reason we don't want it to carry over?

Reply:
We still need that for this PR, but I will remove it in a subsequent one. Here, we are still using the workflow-tools repository cloned as part of manage_externals.

Once I can replace that logic, I'll also remove this PYTHONPATH stuff.